dojo-dungeon/src/dojo/rooms/rooms/riddle/Fehler_Syntax.java

```diff
@@ -108,7 +108,7 @@ private void generate() throws IOException {
     Function<Task, Boolean> openDialog3 =
         (t) -> {
           DojoCompiler.TestResult results =
-              new DojoCompiler().testWrongClass1(FILENAME2, CLASS_NAME);
+              new DojoCompiler().testWrongClass1(FILENAME1, CLASS_NAME);
```

Comment:
wieso arbeitest du hier eigentlich mit pfaden relativ zum subprojekt und nicht mit asset-pfaden (getresource)?

nur neugierig, da es funktioniert, muss das nicht dringend angepasst werden.

Reply:
Danke fürs Review.

Wir sind aus Sicht der JRE ja im build-Verzeichnis des Subprojekts, deshalb: Gehe eins nach oben und dann in die todo-assets...

Die Studis sind aber im Projekthauptverzeichnis und "../" könnte alles durcheinanderbringen.

Reply:
für die studis ist klar, da musst du den pfad vom projekt-ordner ausgeben.

aber intern hätte ich irgendwie sowas wie getResources oder so im einlesen erwartet? das geht dann ja automatisch in den assets-ordner?

Reply:
Unser `todo-assets`-Ordner ist eigentlich nur ein Fake-Resource-Order, :D das heißt, `getResources` würde den nicht aufgreifen... Die build gradle ändern geht auch nicht, weil dann die `todo-assets` auch von der CI erfasst würden... hm